### PR TITLE
7.0 Fixed page breaks

### PR DESCRIPTION
**Pull request by llacroix**
Module level blocks aren't executed more than once. If you
want to use python code in your template in a for loop for
example. You should use <% %> instead of <%! %>.

### CODE REVIEW

File: account_partner_aged_statement_webkit/report/partner_aged_statement.mako [hunk 1]
```diff
@@ -21,14 +21,13 @@
...
-
-    first = True
     %>
+    <% first = True %>
     <body>
     %for partner in objects:
         <%setLang(partner.lang)%>
         %if not first:
-            <div class="break"></div>
+            <div class="break"></div>
```
Comment (bwrsandman) [inline account_partner_aged_statement_webkit/report/partner_aged_statement.mako:30]: Unnecessary change

File: account_partner_aged_statement_webkit/report/partner_aged_statement.mako [hunk 2]
```diff
@@ -21,14 +21,13 @@
     def amount(text):
         # replace by a non-breaking hyphen (it will not word-wrap between hyphen and numbers)
         return text.replace('-', '&#8209;')
-
-    first = True
     %>
+    <% first = True %>
     <body>
     %for partner in objects:
```
Comment (bwrsandman) [inline account_partner_aged_statement_webkit/report/partner_aged_statement.mako:27]: Wouldn't it be cleaner to do:

``` mako
%for i, partner in enumurate(objects):
    %if i > 0:
        <div class="break"></div>
    %endif
```

Then `first` could be removed

Comment (veloutin) [inline account_partner_aged_statement_webkit/report/partner_aged_statement.mako:27]: Yes... fixed upstream.

Comment (llacroix) [inline account_partner_aged_statement_webkit/report/partner_aged_statement.mako:27]: I'll close the PR then.

**Comment by llacroix** [comment]
Fixed upstream no need for merge
